0.5/articles/unit-testing-elements.md

```diff
@@ -269,7 +269,7 @@ Give it an ID so we can easily reference it amongst other instances on the page.
 Next, we'll query the DOM for the "selector1" element we just included included:
 
     <script>
-      var s = document.querySelector('#selector1');
+      var s = document.querySelector('#selector1'); // why not byid?
```

Comment:
We use qSA for IDs pretty regularly. Given that it'll just call out to the best fast path behind the scenes, I think it's a non-issue.